### PR TITLE
[stdlib] Audit inlinability in BridgeObjectiveC [WIP]

### PR DESCRIPTION
**Pull request by moiseev**
<rdar://problem/45949961>

### CODE REVIEW

**Comment by moiseev** [comment]
@swift-ci Please benchmark

**Comment by moiseev** [comment]
@swift-ci Please smoke test

**Comment by moiseev** [comment]
@swift-ci Please benchmark

**Comment by moiseev** [comment]
@swift-ci Please benchmark

**Comment by moiseev** [comment]
@swift-ci Please test

**Comment by moiseev** [comment]
@swift-ci Please benchmark

File: stdlib/public/core/BridgeObjectiveC.swift [hunk 1]
```diff
@@ -166,16 +166,17 @@ public func _bridgeAnythingToObjectiveC<T>(_ x: T) -> AnyObject {
 }
 
 @_silgen_name("")
```
Comment (jrose-apple) [inline stdlib/public/core/BridgeObjectiveC.swift]: What is this doing?

Comment (moiseev) [reply stdlib/public/core/BridgeObjectiveC.swift]: https://github.com/apple/swift/pull/13005#discussion_r151886972

**Comment by moiseev** [comment]
@swift-ci Please benchmark

**Comment by moiseev** [comment]
@swift-ci Please test

**Comment by moiseev** [comment]
@swift-ci Please test

File: stdlib/public/core/BridgeObjectiveC.swift (hunk 1 above)
Comment (jrose-apple) [reply stdlib/public/core/BridgeObjectiveC.swift]: Oof, so we're using the usableFromInline-ness but also the ability to have such a function without a body. I guess that can't change for now.

**Pull request merged**